client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -14,7 +14,7 @@ const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate,
 	if ( hasDomainCredit && selectedSite.plan.user_is_owner && ! isPressableSite ) {
 		return ( <PurchaseDetail
 				icon="globe"
-				title={ translate( 'Select Your Custom Domain' ) }
+				title={ translate( 'Select your custom domain' ) }
```

Comment:
Warning: 21 translations will be lost with this change.
[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)